src/plugins/search/component.js

```diff
@@ -171,11 +171,6 @@ function bindEvents() {
 
   let timeId;
   // Prevent to Fold sidebar
-  Docsify.dom.on(
```

Comment:
After deletion, the sidebar will be collapsed when using search on the mobile end, making search unavailable.